charmander/main.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
These are thin wrappers on functions that come from the libraries to

a) run the development server
b) set up the datastore, in this case postgres